tgui/packages/tgui/interfaces/TcommsCore.js

```diff
@@ -0,0 +1,304 @@
+import { Fragment } from 'inferno';
+import { useBackend, useLocalState } from "../backend";
+import { Button, LabeledList, Box, Section, NoticeBox, Tabs, Icon, Table } from "../components";
+import { Window } from "../layouts";
+
+export const TcommsCore = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    ion,
+  } = data;
+  const [tabIndex, setTabIndex] = useLocalState(context, 'tabIndex', 0);
+
+  const PickTab = index => {
+    switch (index) {
+      case 0:
+        return <ConfigPage />;
+      case 1:
+        return <LinkagePage />;
+      case 2:
+        return <FilteringPage />;
+      default:
+        return "SOMETHING WENT VERY WRONG PLEASE AHELP";
+    }
+  };
+
+  return (
+    <Window resizable>
+      <Window.Content scrollable>
+        {ion === 1 && (
+          <IonBanner />
+        )}
+        <Tabs>
+          <Tabs.Tab
+            key="ConfigPage"
+            selected={tabIndex === 0}
+            onClick={() => setTabIndex(0)}>
+            <Icon name="wrench" />Configuration
+          </Tabs.Tab>
+          <Tabs.Tab
+            key="LinkagePage"
+            selected={tabIndex === 1}
+            onClick={() => setTabIndex(1)}>
+            <Icon name="link" />Device Linkage
+          </Tabs.Tab>
+          <Tabs.Tab
+            key="FilterPage"
+            selected={tabIndex === 2}
+            onClick={() => setTabIndex(2)}>
+            <Icon name="user-times" />User Filtering
+          </Tabs.Tab>
+        </Tabs>
+        {PickTab(tabIndex)}
+      </Window.Content>
+    </Window>
+  );
+};
+
+const IonBanner = () => {
+  // This entire thing renders on one line
+  // Its just split in here to get past
+  // the 80 char line limit
+  return (
+    <NoticeBox>
+      ERROR: An Ionspheric overload has occured.
```

Comment:
`Ionspheric` -> `Ionospheric`

Reply:
Note to self: Update comment in `_base.dm`